akka-docs/build.sbt

```diff
@@ -22,8 +22,4 @@ paradoxProperties ++= Map(
   "snip.akka.base_dir" -> ((baseDirectory in Test).value / "..").getAbsolutePath
 )
 
-resolvers += Resolver.bintrayRepo("2m", "maven")
```

Comment:
With the default config in akka-paradox the theme was published to sbt-plugin repo. So I did not reconfigure it and kept it as it was.